app/controllers/api/v1/admin_controller.rb

```diff
@@ -67,8 +67,18 @@ def save_facebook_credentials_for_smooch_bot(platform) # "platform" is either "i
     tbi = TeamBotInstallation.find(params[:id])
     auth = session['check.facebook.authdata']
     status = nil
-    if params[:token].to_s.gsub('#_=_', '') == tbi.get_smooch_authorization_token
-      response = Net::HTTP.get_response(URI("https://graph.facebook.com/me/accounts?client_id=#{CheckConfig.get('smooch_facebook_app_id')}&client_secret=#{CheckConfig.get('smooch_facebook_app_secret')}&access_token=#{auth['token']}&limit=100"))
+    if auth.blank?
+      status = 400
+      @message = I18n.t(:invalid_facebook_authdata)
+      CheckSentry.notify('Could not authenticate Facebook account for tipline Messenger integration.')
```

Comment:
Sawy, please add more information to the Sentry notification (not to the message, but as a second hash parameter) so we know to which workspace it refers to (maybe at least the `TeamBotInstallation` ID).